archive/p/python/game_of_life.py

```diff
@@ -26,17 +26,13 @@ def is_alive(self):
         method checks if the cell is ALIVE
         returns True if it is alive, False if not.
         """
-        if self._status == 'Alive':
-            return True
-        return False
+        return True if self._status == 'Alive' else False
```

Comment:
If you're going through the effort to change this, you might as well just return this status directly:

```suggestion
        return self._status == 'Alive'
```